### PR TITLE
Implement installer updates

### PR DESCRIPTION
**Pull request by mcalmer**
## What does this PR change?

Use installer update repos for autoinstallation.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- https://github.com/SUSE/spacewalk/issues/12172

- [x] **DONE**

## Test coverage
- Unit tests were adapted

- [x] **DONE**

## Links

Fixes https://github.com/SUSE/spacewalk/issues/3934

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"		 
- [ ] Re-run test "java_pgsql_tests"		 
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"		 
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"		 
- [ ] Re-run test "spacecmd_unittests" (Test skipped, there are no changes to test)

### CODE REVIEW

**Comment by juliogonzalez** [comment]
Dear developer:

A branch for SUSE Manager 4.1 is now created on the internal repository, and all package SPEC files in the master branch of this repository are now updated to be '4.2.0' version.

All developers, regardless of being SUSE employee or not, need to keep the following things in mind:

1. Please rebase this PR against `master`branch and fix any conflicts.
2. If this PR includes schema upgrades, change the directory for the migration scripts to the **correct one**. At this precise moment it is `schema/spacewalk/upgrade/susemanager-schema-4.2.0-to-susemanager-schema-4.2.1` but it could be a different one if this PR is still open after Uyuni 2020.08 is released.

SUSE employees must also keep in mind:

1. If you merge the PR here, you most likely want to backport it to the new `Manager-4.1` branch.
2. You will need to **adapt** the PR when backporting for at least the changelogs and schema migrations (if there are on your PR).

File: java/code/src/com/redhat/rhn/domain/product/SUSEProductSCCRepository.java [hunk 1]
```diff
@@ -57,6 +57,7 @@
     private String parentChannelLabel;
     private String channelName;
     private boolean mandatory;
+    private boolean installerUpdates = false;
```
Comment (lucidd) [inline java/code/src/com/redhat/rhn/domain/product/SUSEProductSCCRepository.java:60]: i think this makes more sense in the SCCRepository table

Comment (mcalmer) [reply java/code/src/com/redhat/rhn/domain/product/SUSEProductSCCRepository.java]: Right. I moved it.

**Comment by mcalmer** [comment]
@lucidd JFYI - manual test successful

**Pull request merged**